packages/client-sdk-nodejs/src/config/middleware/cache-request-logging.ts

```diff
@@ -110,11 +110,11 @@ interface SetIfRequestLog extends WriteRequestLog {
   value: string;
   condition: string;
   present: boolean;
-  presentAndNotEqual: string | undefined;
+  presentAndNotEqual: string | null;
   absent: boolean;
-  equal: string | undefined;
-  absentOrEqual: string | undefined;
-  notEqual: string | undefined;
+  equal: string | null;
+  absentOrEqual: string | null;
+  notEqual: string | null;
```

Comment:
This is a breaking change if the interface is exposed externally. It should be fine to use either `<field>: string | undefined` or `<field>?: string`.

More generally, guidance should be to always use `undefined` as the type, `null` for comparison, and neither anywhere else. (Sorry, I know this is confusing.)

Reply:
This is not an interface anyone will have written any code against. This seemed like the path of least resistance for fixing this one.